### DISCUSSION TITLE
large miniserv.error

### DISCUSSION THREAD

**Discussion by AlucardGO** [webmin/webmin#2161] [General]
I just start to have this issue with webmin file miniserv.error, it get too large and it takes minutes to fill up to 400gb of data, the only message in this file is: "Port number above 0xFFFF, will be truncated to 4997 for Socket::pack_sockaddr_in at /usr/share/webmin/fastrpc.cgi line 352." I cant find the root cause, just restart webmin solve the problem, but after 1 or 2 days it suddently start again. Someone already get trough this ?

Webmin: 2.111 and some 2.011
Ubuntu server: 22.04.4 LTS
all of them are in a samba4 domain, winbind.

Thanks!

**Comment by jcameron** [comment]
Wow, that's an odd message! Are you connecting to Webmin from an IPv6 address?

**Reply by AlucardGO** [reply]
We dont have IPv6 implemented yet, but those servers still have an IPv6 valid IP. Should I disable IPv6 on our linux servers ?

-- 11574 root      20   0   45844  34676   5892 R 100,0   0,4 107:15.00 **/usr/share/webm**
Above is the process that write all the data

**Reply by jcameron** [reply]
You can turn off IPv6 in the Webmin Configuration module.

**Reply by AlucardGO** [reply]
Just checked right now, already disabled.
![image](https://github.com/webmin/webmin/assets/103897437/b75f7361-b90b-4c03-a563-615cf5c55e72)

**Reply by AlucardGO** [reply]
I just disabled this option now.

![image](https://github.com/webmin/webmin/assets/103897437/32011f7d-8ef9-43f5-8be3-9864ddd05d34)

**Reply by jcameron** [reply]
Let us know if that helps at all ...

**Reply by AlucardGO** [reply]
Disabling “Accept IPv6 connections” appears to have resolved the issue. I haven't had any problems since Friday.

**Reply by jcameron** [reply]
That's interesting but surprising, as the port range for IPv6 is the same as IPv4.

**Comment by Kleofass** [comment]


**Reply by iliaross** [reply]
That's bizarre for `last` command to be missing. Though my Debian 13 has it.

Yours doesn't!?

**Reply by jcameron** [reply]
@iliaross maybe we should check if the `last` command is installed before calling it?

**Comment by iliaross** [comment]
> @iliaross maybe we should check if the last command is installed before calling it?

No, I don’t think we should because last is an essential command, and it's better to let the user know that it's missing rather than hiding it. It's not supposed to be missing, so if it is, the user should fix their system instead.

**Reply by jcameron** [reply]
What do we use it for though?

**Reply by iliaross** [reply]
We use it in many places, like to "View Login Sessions".

> No, mine didn't have it, not even a symbolic link. I was sure it was only in the util-linux package, but I found the wtmpdb package, which has /usr/bin/last for Debian 13.
I installed it and the error doesn't occur anymore.

Does your system provide a meta package for the `last` command? I mean, if you run `apt-get install last`, will it install the right package for it?

**Reply by iliaross** [reply]
We can't work around custom-made builds; I think you should just install the `last` command, and the problem is solved.

**Reply by jcameron** [reply]
I checked in https://github.com/webmin/webmin/commit/37f1b5239126815df99a41a8d944cedbbbadbca9 to at least deal with a missing `last` command more gracefully.
